crates/bin/pindexer/src/dex_ex/mod.rs

```diff
@@ -668,6 +672,40 @@ struct PairMetrics {
     liquidity_change: f64,
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize)]
+struct BatchSwapSummary {
+    #[serde(serialize_with = "serialize_asset_id")]
+    asset_start: penumbra_sdk_asset::asset::Id,
+    #[serde(serialize_with = "serialize_asset_id")]
+    asset_end: penumbra_sdk_asset::asset::Id,
```

Comment:
We could just use `Vec<u8>` and sqlx's `BigDecimal`, that way we do not need to marshal the struct to json.